python/neuroglancer/ingest/base.py

```diff
@@ -1,51 +1,72 @@
 import string
-import random
 import os
 import shutil
+import tempfile
 import subprocess
 from google.cloud import storage
 
-BUCKET_NAME = 'neuroglancer'
-QUEUE_NAME = 'pull-queue'
-PROJECT_NAME = 'neuromancer-seung-import'
+import lib
```

Comment:
I don't own any of the code, feel free to modify all of it.
My intuition is that sometime in the future we might want to support s3 or a local filesystem.
That's why I imagine the storage class being able to pull and push data base on the protocol (s3://,gs://,file://)